configs/ttalps_skimmer_signalLike_semimuonic_config.py

```diff
@@ -11,6 +11,22 @@
 applySignalLikeSkimming = True
 
 weightsBranchName = "genWeight"
+eventsTreeNames = ("Events",)
+
+# For the signal like skimming all given muonMatchingParams are applied together
+# If only one matching method should be used ONLY include that one method
+# We might want to update the logic of this in the future
+# Matching methods implemented are:
+# "Segment" : max matching ratio (eg. 2.0/3.0)
+# "DR" : max Delta R (eg. 0.1)
+# "OuterDR" : max Delta R (eg. 0.1)
+# "ProxDR" : max Delta R (eg. 0.1)
+muonMatchingParams = {
```

Comment:
I changed the way I'm choosing the muon matching methods now. 
Right now I'm using muonMatchingParams for both ttalps_skimmer and ttalps_histogrammer BUT they are used in different ways. Because for the skimmer it leaves the option to apply more than one matching method, for example like in displaced muons where they have both Segment and ProxDR.

I think this can be redone in an even nicer way, but at least now all histograms will only have to be declared once and we can easily just change it here.

Reply:
This might be unnecessary for us later when we decide on the matching methods we will use